doc/state_estimation_nodes.rst

```diff
@@ -150,6 +150,14 @@ If fusing accelerometer data from IMUs, this parameter determines whether or not
 
 .. note:: This assumes that the IMU that is providing the acceleration data is also producing an absolute orientation. The orientation data is required to correctly remove gravitational acceleration.
 
+~imuN_dynamic_magnetometer_correction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If using data from one EKF to correct the magnetometer input on another EKF, this set of parameters will define that.
+imuN_dynamic_magnetometer_correction: The topic that the EKF should listen to for data from the uncorrected EKF
+imuN_dynamic_magnetometer_correction_min_speed: Minimum speed for dat to be used
```

Comment:
```suggestion
imuN_dynamic_magnetometer_correction_min_speed: Minimum speed for data to be used
```

Reply:
Thank you. Have updated in the commit below.